### PR TITLE
Fix WNPRC_PurchasingTest.testEmailCustomizationLineItemUpdate

### PR DESCRIPTION
**Pull request by XingY**
#### Rationale
ehr_purchasing.purchasingRequests has a custom query definition in wnprc_purchusing that adds special calculated columns. The email notifications sent are dependent on those calculated columns, which aren't returned by updateRows using prepared statement. This PR flag all ehr_purchasing tables to use the old row-by-row updates, instead of prepared statement.

TC failure:
https://teamcity.labkey.org/buildConfiguration/LabkeyTrunk_EhrPostgres/2274993?buildTab=tests&status=failed

#### Related Pull Requests
* https://github.com/LabKey/ehrModules/pull/529
* https://github.com/LabKey/wnprc-modules/pull/251

#### Changes
* update updateRows parameters to match updated signature

### CODE REVIEW

File: WNPRC_EHR/src/org/labkey/wnprc_ehr/TriggerScriptHelper.java [hunk 1]
```diff
@@ -1548,14 +1554,14 @@ else if (Date.from(endOfLoop.atStartOfDay(ZoneId.systemDefault()).toInstant()).g
                         if(!toUpdate.isEmpty()){
                             Container container = getContainer();
                             User user = getUser();
+                            BatchValidationException errors = new BatchValidationException();
```
Comment (labkey-jeckels) [inline WNPRC_EHR/src/org/labkey/wnprc_ehr/TriggerScriptHelper.java:1557]: This isn't used. Presumably the call to updateRows() on the next line should pass it in?

File: WNPRC_EHR/src/org/labkey/wnprc_ehr/TriggerScriptHelper.java [hunk 2]
```diff
@@ -2169,7 +2175,8 @@ public JSONArray changeWaterScheduled(String animalId, Date startDate, String wa
                 {
                     Container container = getContainer();
                     User user = getUser();
-                    List<Map<String, Object>> updatedRows = waterOrders.getUpdateService().updateRows(user, container, toUpdate, oldKeys, null, null);
+                    BatchValidationException errors = new BatchValidationException();
+                    List<Map<String, Object>> updatedRows = waterOrders.getUpdateService().updateRows(user, container, toUpdate, oldKeys, errors, null, null);
```
Comment (labkey-jeckels) [inline WNPRC_EHR/src/org/labkey/wnprc_ehr/TriggerScriptHelper.java:2179]: Do we not need to check errors.hasErrors() for this usage?

**Review by labkey-jeckels — CHANGES_REQUESTED** [review]

File: WNPRC_EHR/src/org/labkey/wnprc_ehr/TriggerScriptHelper.java (hunk 1 above)
Comment (XingY) [reply WNPRC_EHR/src/org/labkey/wnprc_ehr/TriggerScriptHelper.java:1557]: good catch. fixed

File: WNPRC_EHR/src/org/labkey/wnprc_ehr/TriggerScriptHelper.java (hunk 2 above)
Comment (XingY) [reply WNPRC_EHR/src/org/labkey/wnprc_ehr/TriggerScriptHelper.java:2179]: fixed

**Review by labkey-jeckels — APPROVED** [review]

**Pull request merged**